app/cdap/components/ConnectionList/Components/HeaderCustomTooltip/index.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box } from '@material-ui/core';
+import CustomTooltip from 'components/ConnectionList/Components/CustomTooltip';
+import HeaderCustomTooltipLabel from 'components/ConnectionList/Components/HeaderCustomTooltipLabel';
+import { IHeaderCustomTooltipProps } from 'components/ConnectionList/types';
+import React from 'react';
+
+export default function({ headersRefs, index, tabsData, filteredData }: IHeaderCustomTooltipProps) {
+  return headersRefs?.current[index]?.offsetWidth < headersRefs?.current[index]?.scrollWidth ? (
+    <CustomTooltip title={tabsData[index - 1].selectedTab} arrow>
+      <Box>
+        <HeaderCustomTooltipLabel
```

Comment:
HeaderCustomTooltipLabel
why do we need custom tooltip keywords in this component name, i think it is independent as tooltip is taken higher the order and is not mandatory for this component

also, try to reuse a variation of render label if possible

thank you

Reply:
Done